gen/dcos-config.yaml

```diff
@@ -109,17 +182,44 @@ package:
   - path: /etc/mesos-master-provider
     content: |
       MESOS_CLUSTER={{ cluster_name }}
-  - path: /etc/mesos-slave-modules.json
+  {% switch dcos_overlay_enable %}
+  {% case "true" %}
+  - path: /etc/mesos-slave-modules/overlay_slave_modules.json
```

Comment:
Any reason not to put both mesos-master-modules, and mesos-slave modules config in the same switch statement?